Rename "File Icons Extended" to "zz File Icons"

oivva opened the pull request.

wbond commented:
Why? Having a package start with `zz` sound suspiciously like some sort of hacky solution.

oivva commented:
Yes, that's hacky solution :wink: I need this to override theme icons if they are provided. If custom theme has support of this package (simply have icons with the same names) it does nothing or it can force override. See [package settings](https://github.com/oivva/st-file-icons/blob/dev/File%20Icons.sublime-settings).

P.S. Working on `file-icons` package which adds functionality like [atom's file icons](https://github.com/DanBrooker/file-icons).

oivva commented:
@wbond I've updated the readme and added [explanation](https://github.com/oivva/zz-file-icons#how-it-works) section to describe how it works. Currently I've added only half of the icons, however I'll finish it at the end of this week. So I can say it ready to rename. I'll publish 2.0.0 right after renaming.

equinusocio commented:
Any update? we all need this package :)

equinusocio commented:
Ping @wbond @FichteFoll :)

equinusocio commented:
1 month for a PR.. :(

equinusocio commented:
@oivva I think i will restore old icons support, and i will release material theme 4 until this PR is merged.

oivva commented:
@equinusocio OK, I'll message you when it will be released :wink:

FichteFoll commented:
>PC needs a better review system :(

I agree, but it's still some time off until that will happen.

I'll see if I can get the easier to review PRs done this weekend (namely package renames and ownership changes), so that the technically already reviewd packages will get the changes applied sooner rather than later.
Package additions will most likely have to wait for a little while longer.

Since this is only a rename, I'll go ahead an merge it right now (even though you mentioned me).

equinusocio commented:
Thanks!!

gnowland commented:
Hooray! 🙌
On Fri, Oct 28, 2016 at 2:09 PM Mattia Astorino <notifications@github.com>
wrote:

> Thanks!!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wbond/package_control_channel/pull/5852#issuecomment-257002901>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEOZJ2stjRiTW0a9nKOAJPOT57_NnQ1wks5q4kebgaJpZM4KMlLo>
> .
>
-- 
--
*Gifford Nowland*
(302) 898-4597 | gnowland@gmail.com
http://www.giffordnowland.com
